### ISSUE TITLE
Composer issue only in Tinkerwell

### ISSUE THREAD

**Issue by yoeriboven** [beyondcode/tinkerwell-community#326]
Tinkerwell returns this when I run my app:

```
Composer detected issues in your platform:

Your Composer dependencies require the following PHP extensions to be installed: pcntl
```

The only thing that changed is I added Laravel Horizon.

The app works perfectly fine on web and when using `php artisan tinker`.

[Here is an issue](https://github.com/composer/composer/issues/8828) on Composer. Not sure if I should open an issue there or here since it only happens in Tinkerwell.

**Comment by mpociot** [comment]
Could you verify that you have selected the correct php version in the Tinkerwell settings?

**Comment by yoeriboven** [comment]
Where can I change the settings?

In the preferences pane I can only change the PHP binary which is set to `/usr/bin/php`.
I'm on Tinkerwell 2.10.2.

**Comment by mpociot** [comment]
Yeah you probably need to set it to /usr/local/bin/php if you installed PHP via homebrew

**Comment by yoeriboven** [comment]
Thanks! That fixed it. 

This also closes #325

**Comment by mpociot** [comment]
Awesome 👏
